docs/reference/release-notes/860.md

```diff
@@ -19,6 +19,137 @@ These release notes identify the new features included in 8.6, including [alpha
 | ---------------------- | ---------------------------- | ------------ | ------------ | --------------------------------------------------------------------------------------- |
 | 8 October 2024         | 14 April 2026                |              |              | [Self-Managed update guide](self-managed/operational-guides/update-guide/850-to-860.md) |
 
+<!-- https://github.com/camunda/product-hub/issues/2377 -->
+
+### Advanced SaaS offering <span class="badge badge--long" title="This feature affects SaaS">SaaS</span> <span class="badge badge--long" title="This feature affects Console">Console</span>
```

Comment:
Nitpick - can you add the URL under the heading?

Reply:
fixed

Reply:
Thank you - apologies!